### PR TITLE
Fix flake8 (and therefore Travis)

### PR DESCRIPTION
**Pull request by PlasmaPower**

### CODE REVIEW

**Review by pchauncey — APPROVED** [review]

**Comment by PlasmaPower** [comment]
@pchauncey Friendly reminder that after #377 passes, reviews no longer also count as votes, so you might want to upvote as well.

**Comment by pchauncey** [comment]
@PlasmaPower Thanks, upvoted

File: github_api/misc.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 from math import exp
 
 
-
```
Comment (flibustier) [inline github_api/misc.py:4]: His talents as a developer are undeniable. This is by far the most beautiful line of code I've seen in my life!

Comment (phil-r) [reply github_api/misc.py:4]: it's basically a lack of line..

Comment (PlasmaPower) [reply github_api/misc.py:4]: You know what they say, the fastest code is no code :P

Comment (rudehn) [reply github_api/misc.py:4]: We sure saved a lot of time parsing that file now 😆

**Pull request merged**